cpp/src/io/json/write_json.cu

```diff
@@ -0,0 +1,733 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file write_json.cu
+ * @brief cuDF-IO JSON writer implementation
+ */
+
+#include "io/csv/durations.hpp"
```

Comment:
```suggestion
#include <io/csv/durations.hpp>
```
Perhaps this should be moved out of the `csv` directory to a more common src or detail directory